src/datatype.rs

```diff
@@ -339,9 +344,9 @@ pub enum DataRef<'a> {
     /// Float
     Float(f64),
     /// String
-    String(String),
+    String(RichText),
```

Comment:
I'm a bit concerned about the overhead of keeping a `RichText` even if this is not required. Can't we have a `RichText` variant here too?

Reply:
Hmm, I'll have to check again. I had some problems when I tried, but I don't remember what and it might be resolved now? :D I'll come back when I tried

Reply:
Awesome!

Reply:
Was easily possible, nice, added ^^